R/process_sc_all_alarms_telecare.R

```diff
@@ -76,8 +76,6 @@ process_sc_all_alarms_telecare <- function(data, sc_demographics = NULL, write_t
 
   # Deal with episodes which have a package across quarters.
   qtr_merge <- at_full_clean %>%
-    # Use lazy_dt() for faster running of code
-    dtplyr::lazy_dt() %>%
```

Comment:
As I checked, using `data.table::as.data.table()` in Package `data.table` can have the similar efficiency to `lazy_dt()` without triggering any issue. But I am not sure if `data.table::as.data.table()` can be recommended here.